config/_default/menus/main.en.yaml

```diff
@@ -6285,6 +6285,16 @@ menu:
       parent: rum_browser
       identifier: rum_browser_setup
       weight: 101
+    - name: Client Side Setup
+      url: real_user_monitoring/browser/setup/client
+      parent: rum_browser
+      identifier: rum_browser_setup_client
+      weight: 1101
+    - name: Setup
```

Comment:
Should "Setup" be "Server Side Setup," or should "Client Side Setup" be deleted from the nav? It seems strange to have one page in the nav that says "Client Side Setup", and another that just says "Setup."